api/preferences/src/main/kotlin/com/imashnake/animite/api/preferences/PreferencesRepository.kt

```diff
@@ -0,0 +1,21 @@
+package com.imashnake.animite.api.preferences
+
+import androidx.datastore.core.DataStore
+import androidx.datastore.preferences.core.Preferences
+import androidx.datastore.preferences.core.stringPreferencesKey
+import com.imashnake.animite.api.preferences.ext.getValue
+import com.imashnake.animite.api.preferences.ext.setValue
+import javax.inject.Inject
+
+private const val ACCESS_TOKEN = "access_token"
+
+class PreferencesRepository @Inject constructor(
+    private val dataStore: DataStore<Preferences>
+) {
+    private val accessTokenKey = stringPreferencesKey(ACCESS_TOKEN)
```

Comment:
I don't see why ACCESS_TOKEN can't just be inlined here

Reply:
Whoops used it somewhere else before, removed now 👍